var/spack/repos/builtin/packages/intel-oneapi-runtime/package.py

```diff
@@ -19,6 +19,12 @@ class IntelOneapiRuntime(Package):
 
     maintainers("rscohn2")
 
+    # The libraries are redistributable according to intel
+    # license. Need to investigate if the way it is redistributed
+    # meets the conditions of the license. Until then, disable
+    # distribution
+    redistribute(source=False, binary=False)
```

Comment:
I removed the redistribute directive and made it clear what they are getting and pointed to the license. When HPSF forms a binary distribution project, I recommend that you look at the licensing for everything.

Reply:
@tgamblin :point_up:

Reply:
@alalazo: Doesn't this just make it so that the user has to install `intel-oneapi-compilers` to "build" the runtime?  Which is really just copying?

I'm not understanding how `intel-oneapi-runtime` isn't complying with the license (it says we can redistribute those pieces) but if @rscohn2 wants us to confirm I think we should respect that.

Reply:
The piece that we probably need clarification is this one:

> Any distribution of the Redistributables must only be as part of Your Product which must add significant primary functionality different than that of the Redistributables themselves;

which is vague. The redistributed package in Spack is only really there because it's needed as a dependency of *something* but it's a separate package.

and:

> You must only distribute the Redistributables originally provided to You by Intel only in
Executable Code subject to a license agreement that prohibits reverse engineering,
decompiling or disassembling the Redistributables;

Which we'd need to figure out. I think this is really intended for executables that use the redistributables and bundle all of this up together; it's not written in a way that is particularly friendly for external packaging systems like us.

I think we are better off getting something more like what UO has -- a specific agreement for Spack -- than trying to argue that the license covers our case. It's unfortunately vague enough that we would want a specific authorization from Intel anyway.